src/components/BarStack/index.tsx

```diff
@@ -117,45 +132,54 @@ const StyledContainer = styled.div`
   ${StyledBarWrapper}:nth-child(5n+3) {
     ${({ theme }) => `background: linear-gradient(-45deg, ${transparentize(
       0.9,
-      theme.colorsDeprecated.white,
+      theme.colors.neutral.backgroundWeak,
     )} 25%,
-      ${theme.colorsDeprecated.lightViolet} 25%, ${
-      theme.colorsDeprecated.lightViolet
+      ${theme.colors.secondary.background} 25%, ${
+      theme.colors.secondary.background
```

Comment:
```suggestion
      ${theme.colors.secondary.backgroundStrong} 25%, ${
      theme.colors.secondary.backgroundStrong
```

From now the color is same but later on it could be different 😄